alica_engine/src/engine/planselector/PartialAssignment.cpp

```diff
@@ -94,7 +94,7 @@ void PartialAssignment::prepare(const Plan* p, const TaskAssignmentProblem* prob
     _assignment.clear();
     _assignment.resize(problem->getAgentCount(), -1);
     _cardinalities.clear();
-    _cardinalities.reserve(p->getEntryPoints().size() + s_allowIdling ? 1 : 0);
+    _cardinalities.reserve(p->getEntryPoints().size() + (s_allowIdling ? 1 : 0));
```

Comment:
alica_engine/src/engine/planselector/PartialAssignment.cpp: priority of + is [higher](https://en.cppreference.com/w/cpp/language/operator_precedence) than ternary conditional

Reply:
can remove the conditional operator entirely

Reply:
do +1 all the time?

Reply:
no
`p->getEntryPoints().size() + s_allowIdling)`
or static_cast to std::size_t if we want to make the bool -> std::size_t conversion explicit

Reply:
Technically we could, also c++ [standard](https://eel.is/c++draft/conv.integral#2) has a guarantees for conversion from bool to integer. But this style looks more explicit imho.